test/edgex_redis_share_connection_sink_rule.jmx

```diff
@@ -139,7 +154,7 @@
                   <boolProp name="HTTPArgument.always_encode">false</boolProp>
                   <stringProp name="Argument.value">{&#xd;
   &quot;id&quot;: &quot;rule1&quot;,&#xd;
-  &quot;sql&quot;: &quot;SELECT meta(*) AS edgex_meta, temperature, humidity, humidity*2 as h1 FROM demo WHERE temperature = 20&quot;,&#xd;
+  &quot;sql&quot;: &quot;SELECT temperature, Humidity, meta(*) AS edgex_meta FROM demo WHERE temperature = 72&quot;,&#xd;
```

Comment:
Why change the rule?

Reply:
Since the message published to topic is changed, corresponding rules need to be changed to test its correctness.